HealthcareSystem/Backend/Service/UsersAndWorkingTime/PatientService.cs

```diff
@@ -55,7 +55,10 @@ public bool DeletePatient(string jmbg)
             _deletedPatientRepository.AddPatient(patient);
             return true;
         }
-
+        /// <summary>
+        /// /getting all patients
+        /// </summary>
+        /// <returns>list of active patients</returns>
         public List<Patient> ViewPatients()
         {
             return _activePatientRepository.GetAllPatients();
```

Comment:
Like you said in the documentation, you are getting all patients. Then is it better to have GetAllPatients() method or ViewPatients() ?

Reply:
This methods name will be changed when we refactor all methods regarding user strategy pattern. Right now it is inconvenient to change it but i'll keep it in mind.

Reply:
Alright then. Just keep it in mind.